mobile/src/main/java/org/openhab/habdroid/core/connection/DefaultConnection.java

```diff
@@ -0,0 +1,48 @@
+package org.openhab.habdroid.core.connection;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+public class DefaultConnection extends AbstractConnection {
+    private String username;
+    private String password;
+    private String openHABUrl;
+
+    DefaultConnection(Context ctx, SharedPreferences settings) {
+        super(ctx, settings);
+    }
+
+    DefaultConnection(Context ctx) {
+        super(ctx);
+    }
+
+    @Override
+    public void setUsername(String username) {
+        this.username = username;
```

Comment:
Should recreate or reset the new username to all https clients (sync and asynchronous).

Reply:
Done